wizard.py

```diff
@@ -0,0 +1,59 @@
+import random
+import settings
+import common_fields
+import tools
+import bolus
+
+def wizard(start_time, gluc, carbs, timesteps, params):
```

Comment:
For all of these type-specific sub-modules, a good docstring with info about the params would be invaluable. In particular, what units (or type of value or object, for the `start_time`) is each expected to be in?